Disable the need for `git` and having an actual git checkout by retrieving the
git revision from an environment variable.

--- a/common.ant.xml	2020-02-26 10:59:22.204299472 +0100
+++ b/common.ant.xml	2020-02-26 10:59:16.952299322 +0100
@@ -276,9 +276,7 @@
 
   <macrodef name="gwt.getgitinfo" description="Identifies the GIT info of a workspace">
     <sequential>
-      <exec executable="git" searchpath="true" outputproperty="gwt.gitrev">
-        <arg line="rev-parse --short HEAD"/>
-      </exec>
+	<property name="gwt.gitrev" value="${env.GWT_GITREV}"/>
       <!-- Generally, filtering requires a sentinel file so that changes to git rev will
          be noticed as invalidating the previously-generated filter output.  This property
          names where such a sentinel lives; it is tested with <available/> and created
